runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBrowseView/ProcessList/ProcessList.tsx

```diff
@@ -10,30 +11,35 @@ import useProcessSearch from '#src-app/hooks/useProcessSearch';
 import { useReplaceQueryParams } from '#src-app/hooks/useReplaceQueryParams';
 import ProcessPageProvider from '#src-app/providers/ProcessPage.provider';
 import { useSelector } from '#src-app/store';
+import { getLastParamOfUrl } from '#src-app/views/utils/routerUtils';
 
 import ProcessListHeader from './Header/ProcessList.header';
 import { DefaultPageSize, ProcessListDisplayMode, LOADING_DEBOUNCE } from './ProcessList.utils';
 import GridView from '../GridView';
+import { ProcessesTabs } from '../Header';
 import ProcessTable from '../ProcessTable/ProcessTable';
 
 const ProcessList: VFC = () => {
+    const router = useRouter();
     const { page: processesPage, loading: isStoreLoading } = useSelector((state) => state.process.all);
     const [displayMode, setDisplayMode] = useState(ProcessListDisplayMode.GRID);
     const showLoading = useLoading(isStoreLoading, LOADING_DEBOUNCE);
     const searchParams = useSearchParams();
 
+    const isCollectionsTab = getLastParamOfUrl(router) === ProcessesTabs.COLLECTIONS;
+    const collectionId = isCollectionsTab ? searchParams.get('collectionId') : undefined;
```

Comment:
in IProcessWithFilters interface collectionId is string | null.

Reply:
It won't be passed to getProcessesPageByCollection thunk if it is undefined and only if it will be null or string